src/ssl.c

```diff
@@ -12182,7 +12185,7 @@ WOLFSSL_SESSION* wolfSSL_get_session(WOLFSSL* ssl)
 {
     WOLFSSL_ENTER("SSL_get_session");
     if (ssl)
-        return GetSession(ssl, 0, 1);
+        return GetSession(ssl, NULL, 1);
```

Comment:
OpenSSL has two functions, SSL_get_session() and SSL_get1_session().
SSL_get1_session() returns the session but increased the reference count. Therefore you have to free it,
SSL_get_session() returns the session as a 'reference'. That is, no free needed after.

Reply:
We need wolfSSL_get_session() to match the OpenSSL API and not return a session that needs to be freed.